platform/windows/display_server_windows.cpp

```diff
@@ -571,10 +571,7 @@ void DisplayServerWindows::_thread_fd_monitor(void *p_ud) {
 			}
 		}
 		dir = dir.simplify_path();
-		dir = dir.replace("/", "\\");
-		if (!dir.is_network_share_path() && !dir.begins_with(R"(\\?\)")) {
-			dir = R"(\\?\)" + dir;
-		}
+		dir = dir.trim_prefix(R"(\\?\)").replace("/", "\\");
```

Comment:
This will likely break extended path support (e.g., long paths), something to be aware of.

Reply:
`SHCreateItemFromParsingName` do not have any long path support, so it won't break anything.

Reply:
Ah, missed that function below. Sounds good.